docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -3442,6 +3451,161 @@ the `vrt2` feature. The default vertical advance for these glyphs is the
 horizontal advance of their corresponding target (upright) glyphs. These values
 will also be overridden by `VertAdvanceY` values.
 
+<a name="9.i"></a>
+### 9.i. STAT table
+The STAT table is organized as follows:
+
+```fea
+table STAT {
+    # Elided fallback name
+    # Design axes
+    # Axis values
+} STAT;
+```
+
+#### Elided fallback name
+Elided fallback name must be defined and at most once. It can be
+defined using full name entries or an existing name ID.
+
+```fea
+ElidedFallbackName {
+    name <name spec>;+
+};
+```
+
+The syntax for the individual name string entries is similar to that of the
+name table `nameID` entries (see see §[9.e](#9.e)) - the only difference is
```

Comment:
Oops, didn't catch the double `see see`

Reply:
The typo is an an old paragraph where the text was copied and slightly edited. I’m fixing three 3 occurrences of it.

Reply:
Great, thanks.